conf/messages

```diff
@@ -769,11 +769,16 @@ nextUpdates.propertyIncome            							   = Property income
 nextUpdates.propertyIncome.UK                                      = UK property
 nextUpdates.propertyIncome.Foreign                                 = Foreign property
 nextUpdates.business            							       = Business income
+
 nextUpdates.optOutOneYear-1                                        = You are currently reporting quarterly on a voluntary basis for the {0} to {1} tax year. You can choose to
 nextUpdates.optOutOneYear-2                                        = opt out of quarterly updates and report annually instead.
+nextUpdates.withReportingFrequencyContent.optOutOneYear-1          = Depending on your circumstances, you may be able to
+nextUpdates.withReportingFrequencyContent.optOutOneYear-2          = view and change your reporting frequency.
 
 nextUpdates.optOutMultiYear-1                                      = You are currently reporting quarterly on a voluntary basis. You can choose to
 nextUpdates.optOutMultiYear-2                                      = opt out of quarterly updates and report annually instead.
+nextUpdates.withReportingFrequencyContent.optOutMultiYear-1        = Depending on your circumstances, you may be able to
+nextUpdates.withReportingFrequencyContent.optOutMultiYear-2        = view and change your reporting frequency.
```

Comment:
Currently i'm doing the ATs for this and I noticed that the added messages (Specifically "view and change your reporting frequency") aren't being used within the views you edited

Reply:
Also can you do me a favour and add IDs to the opt in/out links in the reporting frequency page (https://github.com/hmrc/income-tax-view-change-frontend/blob/main/app/views/ReportingFrequencyView.scala.html#L61-L79) so i can get selenium to click them